cmd/lncli/cmd_query_mission_control.go

```diff
@@ -0,0 +1,35 @@
+// +build routerrpc
+
+package main
+
+import (
+	"context"
+
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+
+	"github.com/urfave/cli"
+)
+
+var queryMissionControlCommand = cli.Command{
+	Name:     "querymc",
+	Category: "Payments",
+	Action:   actionDecorator(queryMissionControl),
+}
+
+func queryMissionControl(ctx *cli.Context) error {
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	client := routerrpc.NewRouterClient(conn)
+
+	req := &routerrpc.QueryMissionControlRequest{}
+	rpcCtx := context.Background()
+	snapshot, err := client.QueryMissionControl(rpcCtx, req)
+	if err != nil {
+		return err
+	}
+
+	printRespJSON(snapshot)
```

Comment:
We should have a custom encoding here for the node pubkey, otherwise it shows up as base58.

Reply:
Added custom encoding. But in general I don't like using two types of json encoders. I think we should consider making a tiny fork of `protobuf` to modify `jsonpb` so that it duplicates slice fields with a `Hex` version.